db/migrate/20191127225110_create_feature_flags.rb

```diff
@@ -0,0 +1,11 @@
+class CreateFeatureFlags < ActiveRecord::Migration[6.0]
+  def change
+    create_table :feature_flags do |t|
+      t.string :name
```

Comment:
Consider adding an index to the `name` column as its likely this column is going to be queried frequently. Guessing it should also be a unique index to prevent duplicates.